guides/araujo/FTP Writeup.md

```diff
@@ -0,0 +1,92 @@
+---
+
+
+---
+
+<hr>
+<h2 id="layout-posttitle-how-to-setup-an-ftp-serverauthor-joshua-araujo">layout: post<br>
+title: How to Setup an FTP Server<br>
+author: Joshua Araujo</h2>
+<h3 id="contact">Contact</h3>
+<ul>
+<li>Slack: @Shinpuru on <a href="http://wcscusf.slack.com">wcscusf.slack.com</a></li>
+<li><a href="look-at-me">LinkedIn</a></li>
+<li>Email: <a href="mailto:araujoj@mail.usf.com">araujoj@mail.usf.com</a></li>
+</ul>
+<h3 id="table">Table</h3>
+<ol>
+<li><a href="#id-link-to-section">Prerequisites</a></li>
+<li><a href="#id0">Summary</a></li>
+<li><a href="#id1">Setting up and Installing Windows Server 2003</a></li>
+<li><a href="#id2">Creating and Configuring an FTP Server</a></li>
+</ol>
+<h2 id="prerequisites-a-idid-link-to-sectiona">Prerequisites <a id="id-link-to-section"></a></h2>
+<ol>
+<li>Have your <a href="https://silexone.github.io/guides/nestor/ISPsetup.html">virtual environment</a> configured</li>
+<li>Have the <a href="https://silexone.github.io/guides/william/DNSNotes.html">DNS</a> machine running.</li>
+<li>Have <a href="link-to-guide">pfSense</a> running.</li>
```

Comment:
When this is merged in along with Nestor's and Sharon's guides make another pull request with the new links. For now this is good, just a note for the future.